apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -132,8 +132,7 @@ public static void Initialize(HttpApplication application, string projectId = nu
 
             // Trace does not use monitored resources, only detect the monitored resource if
             // a project id is needed.
-            projectId = projectId ?? CommonUtils.GetAndCheckProjectId(
-                projectId, MonitoredResourceBuilder.FromPlatform());
+            projectId = projectId ?? CommonUtils.GetAndCheckProjectId(projectId);
```

Comment:
No it is an internal class in the Google.Cloud.Diagnostics.Common.

If it's something that we think we will be using we may want to put something similar in Google.Api.Gax.MonitoredResourceBuilder